scripts/upstreamCommit.sh

```diff
@@ -5,7 +5,7 @@ PS1="$"
 
 function changelog() {
     base=$(git ls-tree HEAD $1  | cut -d' ' -f3 | cut -f1)
-    cd $1 && git log --oneline ${base}...HEAD
+    cd $1 && git log --oneline ${base}...HEAD | sed "s/\([0-9a-f]\{8\}\)\s#\([0-9]\+\)/\1 PR-\2/g"
```

Comment:
Can use `'` to simplify. Needs testing by @electronicboy

Reply:
Not sure how you could change quote to simplify that - the issue is sed using BRE which requires escaping most special chars to make them special. We could use ERE, which seems to be in POSIX sed, but I'm not 100% certain.

That being said, there's a few things I'd change:
1. There's no reason for global search, since there won't be more than one of `<COMMIT_ID> #<TICKET>` per line.
2. We can anchor search to beginning of line.

```suggestion
    cd $1 && git log --oneline ${base}...HEAD | sed 's/^\([0-9a-f]\{8\}\)\s#\([0-9]\+\)/\1 PR-\2/'
```

That being said, maybe it should just do global search for all matches for `\s#\([0-9]\+\)`, to also catch reverts etc, like here: https://github.com/PaperMC/Paper/pull/6149

Reply:
Ok, addressed the revert issue by adding a specific check for that.